reference/plugins/plugins_framework.md

```diff
@@ -73,11 +74,12 @@ Extensions to `Praxis::Controller` are done by defining a `Controller` module wi
 
 {% highlight ruby %}
 module Controller
-  # extend AS::Concern so that we can use its included callback to easily register our callback in concrete controllers. Note that Praxis::Controller
-  # also extends AS::Concern.
+  # extend AS::Concern so that we can use its included callback to
+  # easily register our callback in concrete controllers. Note that
+  # Praxis::Controller also extends AS::Concern.
```

Comment:
Updated formatting removes horizontal scrollbar in code examples which improves readablity.